code/datums/station_traits/_station_trait.dm

```diff
@@ -65,9 +65,9 @@
 	qdel(src)
 
 ///Called by decals if they can be colored, to see if we got some cool colors for them. Only takes the first station trait
-/proc/request_station_colors(atom/thing_to_color, pattern = PATTERN_DEFAULT)
+/proc/request_station_colors(atom/thing_to_color, pattern)
 	for(var/datum/station_trait/trait in SSstation.station_traits)
-		var/decal_color = trait.get_decal_color(thing_to_color, pattern)
+		var/decal_color = trait.get_decal_color(thing_to_color, pattern || PATTERN_DEFAULT)
```

Comment:
is this not doing the same thing?

Reply:
Yes, but it also ensures that the `pattern arg` for the `get_decal_color` proccall defaults to `PATTERN_DEFAULT` when null. See L53 of the decal file.